src/utils/helpers/ObjectHelpers.ts

```diff
@@ -0,0 +1,4 @@
+/** A wrapper for Object.prototype.hasOwnProperty to avoid calling it from objects directly (lint rule). */
+export function hasOwnProp<T extends object>(obj: T, prop: string): prop is keyof T & string {
+  return Object.prototype.hasOwnProperty.call(obj, prop);
```

Comment:
`Object.hasOwn` is the preferred replacement. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) It eliminates the need for this helper method. It has wide support, and if it isn't wide enough yet, Babel will transpile it for us.

Reply:
Done